acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-order-line-exchange-rate-update.feature

```diff
@@ -0,0 +1,237 @@
+# For MODORDERS-1185
+Feature: Check encumbrances after order line exchange rate update
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * call loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * call loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def createInvoice = read('classpath:thunderjet/mod-invoice/reusable/create-invoice.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+  @Positive
+  Scenario: Check encumbrances after order line exchange rate update to manual rate 2 times
+    * def fundId = call uuid
+    * def budgetId = call uuid
+    * def orderId = call uuid
+    * def poLineId = call uuid
+    * def invoiceId = call uuid
+    * def invoiceLineId = call uuid
+
+    ### 1. Create finances
+    * configure headers = headersAdmin
+    * def v = call createFund { id: "#(fundId)" }
+    * def v = call createBudget { id: "#(budgetId)", allocated: 10000, fundId: "#(fundId)", status: "Active" }
+    * configure headers = headersUser
+
+    ### 2. Create an order and line
+    * def v = call createOrder { id: "#(orderId)" }
+    * def poLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+    * set poLine.id = poLineId
+    * set poLine.purchaseOrderId = orderId
+    * set poLine.fundDistribution[0].fundId = fundId
+    * set poLine.fundDistribution[0].code = fundId
+    * set poLine.cost.listUnitPrice = karate.get('listUnitPrice', 1.0)
+    * set poLine.cost.poLineEstimatedPrice = karate.get('listUnitPrice', 1.0)
+    * set poLine.isPackage = karate.get('isPackage', false)
+    * set poLine.titleOrPackage = karate.get('titleOrPackage', 'test')
+    * set poLine.paymentStatus = 'Pending'
+    Given path 'orders/order-lines'
+    And request poLine
+    When method POST
+    Then status 201
```

Comment:
we can change these lines to this
```
* def v = call createOrderLine { id: '#(poLineId)', orderId: '#(orderId)', fundId: '#(fundId)' }
```

Reply:
these lines also can be changed https://github.com/folio-org/folio-integration-tests/pull/1534/files#diff-c7ab6f3c8f5642fa693b93d6478fbefafb677baefefdf69ac5112f423c8c173bR148-R162